src/pages/VaccineModulePage.js

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable react/forbid-prop-types, global-require */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Image, StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { Button } from 'react-native-ui-components';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { VaccineChart } from '../widgets/VaccineChart';
+import globalStyles, {
+  SHADOW_BORDER,
+  APP_FONT_FAMILY,
+  SUSSOL_ORANGE,
+  GREY,
+  HAZARD_RED,
+} from '../globalStyles';
+
+const CHERVON_ICON_STYLE = { size: 18, color: SUSSOL_ORANGE };
+const BREACH_ICON_STYLE = { size: 25, color: HAZARD_RED };
+
+// TODO navigation for menu buttons
+// TODO Localise all strings
+
+export class VaccineModulePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { fridges: [], selectedFridgeCode: null };
+  }
+
+  componentWillMount() {
+    const { database } = this.props;
+    const fridges = database.objects('Location').filter(({ isFridge }) => isFridge);
+    const selectedFridgeCode = fridges.length > 0 && fridges[0].code;
+    this.setState({ fridges, selectedFridgeCode });
+  }
+
+  /* Helper to render menuButton */
+  renderMenuButton = buttonProps => (
+    <Button
+      style={[globalStyles.menuButton, localStyles.menuButton]}
+      textStyle={globalStyles.menuButtonText}
+      {...buttonProps}
+    />
+  );
+
+  renderIcon = (iconName, iconStyle = CHERVON_ICON_STYLE) => (
+    <Icon style={{ margin: 5 }} name={iconName} {...iconStyle} />
+  );
+
+  /* Render fridge name and chevron-down icon if icon not selected fridge */
+  renderFridgeName = (fridge, isFridgeSNotelected) => {
+    const onPress = () => this.setState({ selectedFridgeCode: fridge.code });
+
+    const { fridgeInfoSectionStyle, fridgeNameTextStyle } = localStyles;
+
+    return (
+      <TouchableOpacity style={fridgeInfoSectionStyle} onPress={onPress}>
+        {!isFridgeSNotelected && this.renderIcon('chevron-down')}
```

Comment:
`!isFridgeSNotelected` here and in parameter

Reply:
thanks, fixed: https://github.com/openmsupply/mobile/pull/875/commits/a32c28829d2380513e8308cc62d52cdf30b32ed9